server/db/db.go

```diff
@@ -0,0 +1,27 @@
+package db
+
+import (
+	"context"
+	"os"
+
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var Ctx = context.TODO()
+
+func DB() *mongo.Database {
+	clientOptions := options.Client().ApplyURI(os.Getenv("MONGODB_URI"))
+
+	client, err := mongo.Connect(Ctx, clientOptions)
+
+	//check for the connection
+	err = client.Ping(Ctx, nil)
+
+	if err != nil {
+		return nil
```

Comment:
Return nil here will cause `NilPointerException` and crash the server.

This func should return an error for its caller to handle the error properly.

Ex:

```go
func DB() (*mongo.Database, error) {
  //...
}
```